### ISSUE TITLE
This is referenced in the code but the db migrations do not create this table

### ISSUE THREAD

**Issue by u8sand** [MaayanLab/GeneRanger#39]
https://github.com/MaayanLab/GeneRanger/blob/7ba967ae516dcf1b61b9093b75a982aab1d2f907/pages/transcript/%5Btranscript%5D.js#L54

**Comment by u8sand** [comment]
Since the dbs are shared, it's possible we should just have another repo for the db/ETL separate from the two webservers..

**Comment by giacomomarino** [comment]
Yeah, I agree that might be the easiest solution. I updated the ETL with the transcript schema and ingestion script for now.
